src/types/homepage.ts

```diff
@@ -112,7 +113,7 @@ export interface EditorHomepageState {
 
 export interface EditorHeroDropdownSection {
   dropdown: {
-    options: []
+    options: { title: string; url: string }[]
```

Comment:
I think title and url here might be optional!

Reply:
updated typing